uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Import/Helpers/DataImportTrait.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
+use CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChange as LeaveBalanceChange;
+use CRM_HRLeaveAndAbsences_Service_LeaveRequestComment as LeaveRequestCommentService;
+use CRM_HRLeaveAndAbsences_BAO_AbsenceType as AbsenceType;
+
+trait CRM_HRLeaveAndAbsences_Import_Helpers_DataImportTrait {
```

Comment:
Except for the methods used to fetch option values, all the others are specific about the import process and I doubt they can be reused anywhere else. In this case, I think it's better to remove the trait and add the method to the parser.